### ISSUE TITLE
Show exit codes in two-line mode

### ISSUE THREAD

**Issue by Bilge** [romkatv/powerlevel10k#2680]
Exit codes are my favourite feature of P10K.

![image](https://github.com/romkatv/powerlevel10k/assets/470626/15308a53-b5a0-4fa3-9c1a-e3695d0094d7)

But I want to use them in two-line mode. In two-line mode, they are hidden. Might I suggest making this a separate option?

**Comment by romkatv** [comment]
You can change it by editing `~/.p10k.zsh`: https://github.com/romkatv/powerlevel10k/blob/16e58484262de745723ed114e09217094655eaaa/config/p10k-rainbow.zsh#L526

**Comment by Bilge** [comment]
Will this not get overwritten every time I run the `configure` wizard?

**Comment by romkatv** [comment]
It will.

**Comment by Bilge** [comment]
Being as that's the case, do you not think my suggestion to include this as a formal optional in the configuration wizard to be a worthwhile addition?

**Comment by romkatv** [comment]
Some software projects give you a single default config that you can edit. `p10k configure` gives you a choice of a starting point, but then you are still supposed to edit the config if the defaults don't do exactly what you want.

**Comment by Bilge** [comment]
Yes, I've also used Starship, but that is quite a different (and somewhat inferior) new user experience. With P10K, rather than just use `configure` as a starting point, I used it to rapidly prototype different combinations of styles, so I did not just run it once as you imagined.

The point here is that I did not suggest a screen for every option, but rather, to respect that this is a unique situation where an option that _is already configurable_ with the wizard is unfortunately both hidden and conflated with another option. The purpose of giving it its own page is to expose a (rather significant) option that, if one were to never choose single-line mode, may never discover even exists.
